src/RemoteLRS.php

```diff
@@ -901,6 +903,27 @@ public function deleteActivityProfile($activity, $id) {
         return $response;
     }
 
+    public function retrieveActivity($activityid) {
+        $response = $this->sendRequest(
+            'GET',
+            'activities',
+            array(
+                'params' => array(
+                    'activityId' => $activityid,
+                ),
+                'headers' => array(
+                    'Accept-language: ' . $_SERVER['HTTP_ACCEPT_LANGUAGE'] . ', *'
+                )
+            )
+        );
+
+        if ($response->success) {
+            $response->content = new \TinCan\Activity(json_decode($response->content, true));
```

Comment:
Shouldn't need to use full name resolution, IOW `\TinCan\Activity` inside this file can just be `Activity` I think.